lib/pages/athlete/components/buttons.dart

```diff
@@ -8,12 +8,32 @@ import 'package:ax_dapp/pages/scout/dialogs/athlete_page_dialogs.dart';
 import 'package:ax_dapp/pages/scout/models/athlete_scout_model.dart';
 import 'package:ax_dapp/repositories/subgraph/usecases/get_buy_info_use_case.dart';
 import 'package:ax_dapp/repositories/subgraph/usecases/get_sell_info_use_case.dart';
+import 'package:ax_dapp/service/controller/controller.dart';
 import 'package:ax_dapp/service/controller/usecases/get_max_token_input_use_case.dart';
 import 'package:ax_dapp/util/athlete_page_format_helper.dart';
 import 'package:ax_dapp/util/colors.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:get/get.dart';
+import 'package:motion_toast/motion_toast.dart';
+import 'package:motion_toast/resources/arrays.dart';
+
+void showWarningToast(BuildContext context) {
+  MotionToast(
+    icon: Icons.warning,
+    primaryColor: Colors.black.withOpacity(0.3),
+    secondaryColor: Colors.amber[400],
+    backgroundType: BackgroundType.solid,
+    title: Text(
+      'Wallet Warning',
+      style: TextStyle(color: Colors.amber[400], fontSize: 18),
+    ),
+    description: Text(
+      'You need to connect Wallet to use this feature.',
+      style: TextStyle(color: Colors.amber[400]),
+    ),
+  ).show(context);
+}
```

Comment:
Please move this for time being into utils and make it an extension method on `BuildContext`.

Reply:
I made a extension method on utils